tests/Http/UserTest.php

```diff
@@ -1215,4 +1215,55 @@ public function testSmsSubscriptionTopicsAreClearedWhenUnsubscribing()
             'sms_subscription_topics' => null,
         ]);
     }
+
+    /**
+     * Test that breakdown badge is added is true after adding news as a topic on a new user.
+     *
+     * @return void
+     */
+    public function testNewsletterBadgeWhenNewUserAddsNewsSubscription()
+    {
+        $newsUser = factory(User::class)
+            ->states('email-subscribed-news')
+            ->create();
+
+        $this->assertMongoDatabaseHas('users', [
+            '_id' => $newsUser->id,
+            'email_subscription_status' => true,
+        ]);
+
+        $user = $newsUser->fresh();
+        $this->assertEquals(['breakdown'], $user->badges);
```

Comment:
What if we combined these into the `assertMongoDatabaseHas` above?

```suggestion
        $this->assertMongoDatabaseHas('users', [
            '_id' => $newsUser->id,
            'email_subscription_status' => true,
            'badges' => ['breakdown'],
        ]);
```